ansible/provision-and-execute-tests/playbook.yml

```diff
@@ -56,4 +59,82 @@
+  hosts: windows
+  vars:
+    node_version: 16.3.0
+
+    # The following information is populated using the gathered inventory variables
+    fluent_bit_package_name: "{{ tags.fb_package_name }}"
+    pr_number: "{{ tags.pr_number }}"
+  environment:
+    NEW_RELIC_API_KEY: "{{ lookup('ansible.builtin.env', 'NEW_RELIC_API_KEY') }}"
+    NEW_RELIC_ACCOUNT_ID: "{{ lookup('ansible.builtin.env', 'NEW_RELIC_ACCOUNT_ID') }}"
+    NEW_RELIC_REGION: "{{ lookup('ansible.builtin.env', 'NEW_RELIC_REGION') }}"
+  tasks:
+    - name: Wait for connection to be available
+      wait_for_connection:
+
+    - name: Install Infrastructure Agent
+      ansible.builtin.include_role:
+        name: newrelic.newrelic_install
+        # Despite being the default, I want to emphasize that we do NOT want the role variables to be
+        # exposed to the play. Otherwise, the "tags" variable specified below would overwrite the AWS "tags"
+        # variable object (that comes from the inventory), which would cause that the `fluent_bit_package_name`
+        # (or any variable depending on the AWS tags) would not be resolvable. Note that this option is only
+        # available in "include_role", and not in "import_role".
+        public: false
+      vars:
+        targets:
+          - infrastructure
+        tags:
+          product: logging
+          owning_team: logging
+          project: fluent-bit-packaging-and-testing
+
+    - name: Configure log forwarding
+      ansible.builtin.include_role:
+        name: create_logging_configs
+      vars:
+        monitored_file: /path/to/file.log
+        monitored_tcp_port: 9012
+        monitored_windows_log_name_using_winlog: logtar_channel
+        monitored_windows_log_name_using_winevtlog: logtar_channel
+
+    - name: Install Fluent Bit package to be tested for this distro
+      ansible.builtin.include_role:
+        name: install_fluent_bit_from_gh_prerelease
+      vars:
+        fb_package_name: "{{ fluent_bit_package_name }}"
+        gh_prerelease_tag: "tmp-pr-{{ pr_number }}"
+      when: pr_number is not regex('^local-.*')
+
+    - name: Install NVM
+      win_chocolatey:
+        name: nvm
+        state: present
+
+    # Ref: https://github.com/chocolatey/chocolatey-ansible/issues/22#issuecomment-680105708
+    - name: Reboot the machine so Chocolatey path will be picked up
+      ansible.windows.win_reboot:
+
+    - name: Install Node JS
+      ansible.windows.win_command: "nvm install {{ node_version }}"
+
+    - name: Run NPM version
+      ansible.windows.win_shell: "nvm use {{ node_version }} ; npm -v"
+      register: npm_version_out
+
+    - name: NPM version result (to be removed)
+      debug:
+        var: npm_version_out
+
+    - name: Run Node version
+      ansible.windows.win_shell: "nvm use {{ node_version }} ; node -v"
+      register: node_version_out
+
+    - name: Node version result (to be removed)
+      debug:
+        var: node_version_out
```

Comment:
All these lines can be removed in the next PR, they are just left here to ensure Node and NPM is correctly installed.